packages/embark/src/lib/modules/blockchain_listener/index.js

```diff
@@ -48,47 +56,64 @@ class BlockchainListener {
 
   _registerConsoleCommands() {
     this.embark.registerConsoleCommand({
-      description: 'Toggles regular transactions used to prevent transactions from getting stuck when using Geth and Metamask',
-      matches: ['regularTxs on', 'regularTxs off'],
-      usage: "regularTxs on/off",
+      description: __('Toggles regular transactions used to prevent transactions from getting stuck when using Geth and Metamask'),
+      matches: ['devtxs on', 'devtxs off', 'regularTxs on', 'regularTxs off'],
+      usage: "devtxs on/off",
       process: (cmd, callback) => {
-        const eventCmd = `regularTxs:${cmd.trim().endsWith('on') ? 'start' : 'stop'}`;
-        this.events.request(eventCmd, callback);
+        if (cmd.startsWith('regularTxs')) {
+          this.logger.info(__("Deprecation notice: The command 'regularTxs on/off' is now deprecated in favor of 'devtxs on/off' and will be removed in future versions."));
+        }
+        const enable = cmd.trim().endsWith('on');
+        this.logger.info(`${enable ? 'Enabling' :  'Disabling'} regular transactions...`);
+        if(enable) {
+          return this._startRegularTxs(() => {
+            const message = __('Regular transactions have been enabled');
+            this.logger.info(message);
+            callback(null, message);
+          });
+        }
+        this._stopRegularTxs(() => {
+          const message = __('Regular transactions have been disabled');
+          this.logger.info(message);
+          callback(null, message);
+        });
       }
     });
-  }
 
-  _registerApiEndpoint() {
-    this.embark.registerAPICall(
-      'get',
-      '/embark-api/regular-txs',
-      (req, res) => {
-        if(!req.query.mode || !['on', 'off'].includes(req.query.mode)) {
-          return res.status(400).send("Invalid parameter 'mode' provided. Must be one of: ['on', 'off']");
-        }
-        this.events.request(`regularTxs:${req.query.mode === 'on' ? 'start' : 'stop'}`, (err, result) => {
-          if(err) {
-            return res.send({ error: err.message });
-          }
-          res.send(result);
+    this.embark.registerConsoleCommand({
+      description: __('Sends a transaction from default --dev account (generally used if txs are getting stuck in geth in development)'),
+      matches: ['sendtx'],
+      usage: "sendtx",
```

Comment:
No need for `usage` since it matches one to one with `matches`

Reply:
Update, thanks!